terralab/config.py

```diff
@@ -52,5 +56,6 @@ def load_config(
         server_port=int(server_port),
         access_token_file=f'{Path.home()}/{config["LOCAL_STORAGE_PATH"]}/access_token',
         refresh_token_file=f'{Path.home()}/{config["LOCAL_STORAGE_PATH"]}/refresh_token',
-        oauth_token_file=f'{Path.home()}/{config["LOCAL_STORAGE_PATH"]}/oauth_token',
```

Comment:
is this what we use for that "hidden" login route that we use for e2e test?  i.e. this is just a name change and no functionality change

Reply:
just a name change and no functionality change